### ISSUE TITLE
PR Request for css-cascade-3

### ISSUE THREAD

**Issue by fantasai** [w3c/transitions#264]
# Document title, URLs, estimated publication date

CSS Cascading and Inheritance Level 3
https://www.w3.org/TR/css-cascade-3/

# Abstract
https://www.w3.org/TR/css-cascade-3/#abstract

# Status
https://www.w3.org/TR/css-cascade-3/#status

# Link to group's decision to request transition
https://lists.w3.org/Archives/Public/www-style/2020Sep/0001.html

# Changes
https://www.w3.org/TR/css-cascade-3/#changes

# Requirements satisfied
yes

# Dependencies met (or not)
Only depends on CSS2 and css-syntax-3. (Normative references list is has too many additional auto-generated references, need to figure out how to trim that down.)

# Wide Review
https://drafts.csswg.org/css-cascade-3/issues-cr-2016

# Issues addressed
https://drafts.csswg.org/css-cascade-3/issues-cr-2016

# Formal Objections
None

# Implementation
https://drafts.csswg.org/css-cascade-3/implementation-report

# Patent disclosures
None

**Comment by swickr** [comment]
Has this specification received horizontal review within the past two years?

If not, please request horizontal review.

**Comment by fantasai** [comment]
@swickr Can you point me to a document detailing the requirements? Which groups exist / how is the request supposed to be formatted / etc.

**Comment by fantasai** [comment]
OK, I found the new "wide review" document (why isn't it linked from the template here? and why is it served off github.io rather than w3.org?). It mentions nothing about two year limit, so if that's a new requirement you might want to document it. https://w3c.github.io/documentreview/

**Comment by plehegar** [comment]
Q: why isn't it linked from the template here?

A: because it would make the template too long. I added a link from
 https://www.w3.org/Guide/transitions?profile=CR&cr=new
and clarified:
  https://www.w3.org/Guide/transitions?profile=PR&cr=new

Q: why is it served off github.io rather than w3.org?

Fixed. Now served from https://www.w3.org/Guide/documentreview 
 Guide has been updated cc @r12a @samuelweiler 

Q: It mentions nothing about two year limit, so if that's a new requirement you might want to document it.

I think we're still wondering on how to address this point properly (is it a process change?). See also the discussion at
  https://lists.w3.org/Archives/Member/w3c-ac-forum/2020JulSep/0019.html

The fact is: it wasn't a change in CSS Cascade that triggers the re-review request, but rather a change in the understanding of Privacy on the Web.

**Comment by fantasai** [comment]
> The fact is: it wasn't a change in CSS Cascade that triggers the re-review request, but rather a change in the understanding of Privacy on the Web.

Then, you should have requested a re-review by Privacy, not everyone. Anyway, I've requested re-review from all the HRGs, and we're blocked, presumably, until they all respond.

**Comment by xfq** [comment]
FYI - I submitted a PR to update the transition guide (but didn't change the issue template): https://github.com/w3c/transitions/pull/263

**Comment by fantasai** [comment]
TAG review https://github.com/w3ctag/design-reviews/issues/555

**Comment by fantasai** [comment]
Privacy Review was [requested 14 September 2020](https://www.w3.org/mid/1be69974-f41f-e294-2af9-5b112358845b@inkedblade.net) and [acknowledged 9 October 2020](https://www.w3.org/mid/684114F2-F2A7-492E-A37A-F65D8C848184@isoc.org). Nothing received since.
Security Review was [requested 14 September 2020](https://www.w3.org/mid/2641eac1-4dad-4f27-ee7b-670606fa6b05@inkedblade.net). No response.
Internationalization Review was [requested 14 September 2020](https://github.com/w3c/i18n-request/issues/130) and [one editorial issue](https://github.com/w3c/csswg-drafts/issues/5637) was filed, closed wontfix, and accepted as such by the commenter.

There's nothing that I as the editor can do at this point other than continue to wait for Privacy and Security to respond. How much longer would you like me to wait?

**Comment by plehegar** [comment]
Diff:
  https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2F2018%2FCR-css-cascade-3-20180828%2F&doc2=https%3A%2F%2Fdrafts.csswg.org%2Fcss-cascade-3%2F

Horizontal comments:
 https://www.w3.org/PM/horizontal/?repo=w3c/i18n-activity#css-cascade

Normative references:
 https://labs.w3.org/normative-references/check?url=https%3A%2F%2Fdrafts.csswg.org%2Fcss-cascade-3%2F

**Comment by swickr** [comment]
Transition approved.

**Comment by svgeesus** [comment]
Publication requested 21 Dec, expected 22 Dec

**Comment by svgeesus** [comment]
[Published 22 Dec](https://www.w3.org/TR/2020/PR-css-cascade-3-20201222/)
